lib/queue_classic_plus/inheritable_attr.rb

```diff
@@ -28,7 +28,7 @@ def self.call(value, uncloneable=uncloneable())
       end
 
       def self.uncloneable
-        [Symbol, TrueClass, FalseClass, NilClass]
+        [Symbol, TrueClass, FalseClass, NilClass, Fixnum]
```

Comment:
BTW this fixes a problem that popped up in the new tests I wrote (unrelated to the code change).